lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -55,11 +55,21 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
       </div>
     </div>
 
+
     <div className="row">
       <div className="col-sm-12">
         <div className="form-group">
           <label htmlFor="subject">{gettext('Subject')}</label>
-          <input type="text" className="form-control" id="subject" />
+          <input type="text" className="form-control" id="subject" value="[Contact Us Form] Request" />
+        </div>
+      </div>
+    </div>
+
+    <div className="row">
+      <div className="col-sm-12">
+        <div className="form-group">
+          <label htmlFor="phone-number">{gettext('Phone Number')}</label>
+          <input type="tel" className="form-control" placeholder="03**-****" id="phone-number"/>
```

Comment:
nit: if we are sure that this application would be used only for the Pakistan and phone number would be always a mobile number instead of landline then we need to make the placeholder to match the format and lenght of mobile numbers in Pakistan `03**-*******`